v3/tasks/20250307-v3-liquidity-bootstrapping-pool/test/task.fork.ts

```diff
@@ -0,0 +1,173 @@
+        paysYieldFees: false,
+      },
+      {
+        token: BAL,
+        tokenType: 0,
+        rateProvider: ZERO_ADDRESS,
+        paysYieldFees: false,
+      },
+    ].sort(function (a, b) {
+      return a.token.toLowerCase().localeCompare(b.token.toLowerCase());
+    });
+  });
+
+  it('has trusted router', async () => {
+    expect(await factory.getTrustedRouter()).to.eq(trustedRouter.address);
+  });
+
+  it('deploys LBP', async () => {
+    startTime = await currentTimestamp();
+
+    const newLBPParams = {
+      owner: admin.address,
+      projectToken: BAL,
+      reserveToken: WETH,
+      projectTokenStartWeight: HIGH_WEIGHT,
+      reserveTokenStartWeight: LOW_WEIGHT,
+      projectTokenEndWeight: LOW_WEIGHT,
+      reserveTokenEndWeight: HIGH_WEIGHT,
+      startTime: startTime.add(HOUR),
+      endTime: startTime.add(DAY),
+      blockProjectTokenSwapsIn: false,
+    };
+
+    const poolCreationReceipt = await (
+      await factory.create('Mock LBP', 'LBP-TEST', newLBPParams, SWAP_FEE, ONES_BYTES32)
+    ).wait();
+
+    const event = expectEvent.inReceipt(poolCreationReceipt, 'PoolCreated');
+    pool = await task.instanceAt(POOL_CONTRACT_NAME, event.args.pool);
+  });
+
+  it('checks pool tokens', async () => {
+    const poolTokens = (await pool.getTokens()).map((token: string) => token.toLowerCase());
+    expect(poolTokens).to.be.deep.eq(tokenConfig.map((config) => config.token.toLowerCase()));
+
+    expect(await pool.getProjectToken()).to.eq(BAL);
+    expect(await pool.getReserveToken()).to.eq(WETH);
+  });
+
+  it('checks pool version', async () => {
+    const version = JSON.parse(await pool.version());
+    expect(version.deployment).to.be.eq(TASK_NAME);
+    expect(version.version).to.be.eq(1);
+    expect(version.name).to.be.eq(POOL_CONTRACT_NAME);
+  });
+
+  it('checks factory version', async () => {
+    const version = JSON.parse(await factory.version());
+    expect(version.deployment).to.be.eq(TASK_NAME);
+    expect(version.version).to.be.eq(1);
+    expect(version.name).to.be.eq(FACTORY_CONTRACT_NAME);
+  });
+
+  it('sale has not started yet', async () => {
+    expect(await pool.isSwapEnabled()).to.be.false;
+  });
+
+  it('initializes the pool', async () => {
+    // Give the admin tokens
+    balToken.connect(whale).transfer(admin.address, INITIAL_BAL);
+    wethToken.connect(whale).transfer(admin.address, INITIAL_WETH);
+
+    await balToken.connect(admin).approve(permit2.address, INITIAL_BAL);
+    await permit2.connect(admin).approve(BAL, trustedRouter.address, INITIAL_BAL, maxUint(48));
+
+    await wethToken.connect(admin).approve(permit2.address, INITIAL_WETH);
+    await permit2.connect(admin).approve(WETH, trustedRouter.address, INITIAL_WETH, maxUint(48));
+
+    await trustedRouter.connect(admin).initialize(
+      pool.address,
+      [BAL, WETH],
+      [INITIAL_BAL, INITIAL_WETH],
+      0,
+      false, // wethIsETH - LBPool doesn't support native ETH
```

Comment:
hmm why not? The router is the one that wraps / unwraps.
```suggestion
      false, // wethIsETH
```